### PR TITLE
feat(JitsiRemoteTrack): Add logs.

### PR DESCRIPTION
**Pull request by hristoterezov**

### CODE REVIEW

File: modules/RTC/JitsiRemoteTrack.js [hunk 1]
```diff
@@ -64,26 +73,35 @@ export default class JitsiRemoteTrack extends JitsiTrack {
         this.muted = muted;
         this.isP2P = isP2P;
 
+        logger.debug(`New remote track added: ${this}`);
```
Comment (saghul) [inline modules/RTC/JitsiRemoteTrack.js]: please don't log full objects like this, mobile suffers a lot

File: modules/RTC/JitsiRemoteTrack.js [hunk 2]
```diff
@@ -9,6 +9,15 @@ const RTCEvents = require('../../service/RTC/RTCEvents');
 let ttfmTrackerAudioAttached = false;
 let ttfmTrackerVideoAttached = false;
 
+/**
+ * List of container events that we are going to process. _onContainerEventHandler will be added as listener to the
+ * container for every event in the list.
+ */
+const containerEvents = [
+    'play', 'pause', 'waiting', 'suspend', 'stalled', 'ratechange', 'playing', 'loadstart',
```
Comment (saghul) [inline modules/RTC/JitsiRemoteTrack.js]: sort alphabetically

File: modules/RTC/JitsiRemoteTrack.js (hunk 1 above)
Comment (hristoterezov) [reply modules/RTC/JitsiRemoteTrack.js]: Hmm... Why? We have `toString`, isn't it used?

Comment (saghul) [reply modules/RTC/JitsiRemoteTrack.js]: Oh, we do? Then yeah, no worries!

**Pull request merged**